src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
`getify.com` is softly reserved in case my company ever had some sort of corporate activity or interest. The OSS work I do (on github) is merely a side project in that respect.

Since this project is not likely to ever have its own site/domain name, and will thus always live on github, I think it's best to style the package namespacing in that way.